silverstripe/comments/SS-2018-015-1.yaml

```
@@ -0,0 +1,14 @@
+title:     'SS-2018-015: Vulnerable dependency'
+link:      https://www.silverstripe.org/download/security-releases/ss-2018-015/
+cve:       ~
+branches:
+    1.x:
+        time:     null
+        versions: ['>=1.3.0', '<1.99.99']
+    2.x:
+        time:     null
+        versions: ['>=2.0.0', '<2.99.99']
```

```suggestion
        versions: ['>=2.0.0', '<3']
```

As per comment above.

---

will change to 2.9.9